boilerplate/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "cloud_based_e-commerce_platform",
+  "version": "1.0.0",
+  "description": "A cloud based E-commerce platform",
+  "author": "Darshit <darshit.chanpura@gmail.com>, Samarth Parikh, Alex Balgavy, Karthik Mynam",
+  "license": "ISC",
+  "repository": "https://github.com/BU-NU-CLOUD-F19/Cloud_Based_E-Commerce_Platform",
+  "homepage": "https://github.com/BU-NU-CLOUD-F19/Cloud_Based_E-Commerce_Platform",
+  "bugs": "",
+  "config": {
+    "appName": "api"
+  },
+  "scripts": {
+    "dev": "nodemon",
+    "start": "node app.js",
+    "lint": "eslint . --fix"
```

Comment:
Please use the lint script I specifed in the [coding guidelines](https://github.com/BU-NU-CLOUD-F19/Cloud_Based_E-Commerce_Platform/blob/master/doc/coding-guidelines.md), this makes sure we keep a consistent style across the whole project.